src/Extensions/AzureBlobStorage/AzureBlobEgressProviderOptions.cs

```diff
@@ -41,6 +41,11 @@ internal sealed partial class AzureBlobEgressProviderOptions
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_AzureBlobEgressProviderOptions_ManagedIdentityClientId))]
         public string ManagedIdentityClientId { get; set; }
 
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_AzureBlobEgressProviderOptions_UseWorkloadIdentity))]
+        public bool UseWorkloadIdentity { get; set; }
```

Comment:
Is this specific enough? Should it be named something like `UseWorkloadIdentityFromEnvironment`? Or if we added other properties in the future to allow specifying the credentials, would this property as need to be set to `true`?

Reply:
I didn't want to do this initially, but thinking about it some more if we add new properties i want to be able to compose them (i.e. bring in some environment variables and some from properties) so I will rename this.